src/lib/stores/accounts.ts

```diff
@@ -132,3 +140,31 @@ export function colorForRelayStatus(status: string): string {
             return "";
     }
 }
+
+export async function hasNostrWalletConnectUri(): Promise<boolean> {
+    try {
+        return await invoke("has_nostr_wallet_connect_uri");
+    } catch (error) {
+        throw new NostrWalletConnectError(`Failed to check NWC URI: ${error}`);
+    }
+}
+
+export async function setNostrWalletConnectUri(uri: string): Promise<void> {
+    if (!uri || !uri.startsWith("nostr+walletconnect://")) {
```

Comment:
What I understand of the NIP 47 docs is that to be a valid uri, the relay and secret query params must be present

Reply:
good catch! added those cases